src/TextOverflow/TextOverflow.tsx

```diff
@@ -33,7 +33,7 @@ export type TextOverflowRequiredProps = {
   /** in case you want to use a custom component (like a `Tooltip`) to render the full content which is passed as the first argument */
   children?: (self: JSX.Element, isOpen?: boolean) => any;
   /** this is the full string */
-  label?: string | number;
+  label: string | number;
```

Comment:
it doesn't make any sense to overflow with an empty tooltip right?

Reply:
I think this is a co-product masked by product. Both `children` and `label` are optional, but you can really only pass one of the two

Reply:
good catch, makes sense, I'd probably update the typings by disallowing `children` then (or the other way around)?

Reply:
can we make it a coproduct?